src/ex2/item_manager.js

```diff
@@ -0,0 +1,53 @@
+import PokemonClient from "./PokemonClient.js";
+
+class ItemManager {
+	constructor() {
+		this.id = 0;
+		this.itemArray = [];
+		this.pokemonClient = new PokemonClient();
+	}
+	addListItem(item) {
+		const { id, value } = this.createId(item);
+		this.itemArray = [...this.itemArray, { id, value }];
+		return this.itemArray;
+	}
+	removeItem(itemId) {
+		this.itemArray = [
+			...this.itemArray.filter((item) => item.id !== parseInt(itemId)),
+		];
+
+		return this.itemArray;
+	}
+
+	async pokemonIdsHendeling(pokeID) {
+		const pokemon = await this.pokemonClient.getPokemon(pokeID);
+		return pokemon;
+	}
+
+	createId(item) {
+		const newItemObj = { id: this.id, value: item };
+		this.id++;
+		return newItemObj;
+	}
+
+	async pokemonIdsTostring(pokemonIdsStr) {
```

Comment:
This function name is not very clear, I would expect a function with this name to return a string but it actually gets a string.
Rename it something that explains better what it does, maybe something like "getManyPokemonItems" or "addMultiplePokemonItems"